### PR TITLE
Fix #11456

### LINKED ISSUE #11456: [MU4 Issue] Preferences > General > Languages - Check for Update button
**Describe the bug**
The check for update button in Preferences > Language is a bit confusing.

The placement of the button suggests that the action is connected to submitting the language selection.

**To Reproduce**
1. Go to Preferences > General > Languages
2. Select language from dropdown
3.Click 'Check for updates'
4. See issue

**Expected behavior**
As the check for updates function is logically separate from the language selection action, it could be clearer if there is more visual distinction.

Maybe this can be moved to the Updates section of Preferences?

**Screenshots**
Screen Shot 2022-04-29 at 12 12 05 PM

**Platform information**
OS: macOS 12.2
Arch.: x86_64
MuseScore version (64-bit): 4.0.0-2243301402
revision: github-musescore-musescore-a6ab311

### PR DESCRIPTION
**Pull request by donarturo11**
Resolves: [*https://github.com/musescore/MuseScore/issues/11456*](https://github.com/musescore/MuseScore/issues/11456)

*(short description of the changes and the motivation to make the changes)*

- [X] I signed [CLA](https://musescore.org/en/cla)
- [X] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [X] I made sure the code compiles on my machine
- [X] I made sure there are no unnecessary changes in the code
- [X] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [X] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [X] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by HemantAntony** [comment]
You should name your commits in the following format: `Fix #11456: Changed text of Check For Updates Button`. Actually that's a bit too large. But I think you got the idea. Similarly your PR should have a better title

File: src/appshell/internal/applicationuiactions.cpp [hunk 1]
```diff
@@ -179,8 +179,8 @@ const UiActionList ApplicationUiActions::m_actions = {
              ),
     UiAction("check-update",
              mu::context::UiCtxAny,
-             QT_TRANSLATE_NOOP("action", "Check for update"),
-             QT_TRANSLATE_NOOP("action", "Check for update")
+             QT_TRANSLATE_NOOP("action", "Check for language updates"),
```
Comment (RomanPudashkin) [inline src/appshell/internal/applicationuiactions.cpp:182]: You must not change this text. This action doesn't relate to languages

File: src/appshell/qml/Preferences/internal/LanguagesSection.qml [hunk 2]
```diff
@@ -74,7 +74,7 @@ BaseSection {
 
             anchors.verticalCenter: parent.verticalCenter
 
-            text: qsTrc("appshell", "Check for update")
+            text: qsTrc("appshell", "Check for language updates")
```
Comment (Jojo-Schmitz) [inline src/appshell/qml/Preferences/internal/LanguagesSection.qml:77]: Rather "translation updates"?

**Comment by Jojo-Schmitz** [comment]
See also #11476

File: src/appshell/qml/Preferences/internal/LanguagesSection.qml (hunk 2 above)
Comment (donarturo11) [reply src/appshell/qml/Preferences/internal/LanguagesSection.qml:77]: And if I well understood, if I would like to change text only, should I modify only this or similar file?

**Pull request closed without merging**